packages/template-parser/tests/index.test.ts

```diff
@@ -15397,4 +15397,20 @@ describe('parseForESLint()', () => {
       `);
     });
   });
+
+  describe('ng-content', () => {
+    it('should support fallback content', () => {
+      expect(
+        parseForESLint(`
+          <ng-content>
+             @if (foo) {
+               <div>Hello!</div>
+             }
+          </ng-content>
+        `, {
+          filePath: './foo.html',
+        }).ast,
+      ).toMatchSnapshot();
```

Comment:
When I tried to use an inline snapshot jest was throwing:
>  Jest: Couldn't locate all inline snapshots.

So I went with a file based snapshot instead. I think given how big these ASTs are, it makes sense to not have them inline anyway

Reply:
Yeah it's weird how that happens sometimes, it seems to be specific to the machine in question

Reply:
I've never been able to resolve it once it starts either...